src/data/search.tsx

```diff
@@ -24,17 +26,34 @@ function indexAkn(html: Document): IndexedObject[] {
     let title = (titleSelector ? titleSelector.textContent : "") || "";
 
     searchData.push({
-      item: {
-        id: section.id,
-        title: title,
-        children: [],
-      },
+      item: '',
+      title: title,
       id: section.id,
       content: title.toLocaleLowerCase() + text.join(' ').toLocaleLowerCase(),
     });
   });
 
-  return searchData;
+  const lunrSearchData = lunr(function () {
+    // @ts-ignore
+    this.ref('id');
+    // @ts-ignore
+    this.ref('title');
+    // @ts-ignore
+    this.ref('content');
+    // @ts-ignore
+    this.field('title');
+    // @ts-ignore
+    this.field('content');
+    // @ts-ignore
+    this.metadataWhitelist = ['position'];
+
+    searchData.forEach(function (doc: {}) {
```

Comment:
You could attach the `searchData` object to the lunr index to help you lookup the items returned by the search later on. eg.

```
this._searchData = searchData
```